py3status/modules/taskwarrior.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display currently active (started) taskwarrior tasks.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds
+
+Requires
+    - jq
+    - awk
+
+@author James Smith http://jazmit.github.io/
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+import subprocess
+
+class Py3status:
+    cache_timeout = 5
+
+    def taskWarrior(self, i3s_output_list, i3s_config):
+        active_task = subprocess.check_output(
```

Comment:
can you please try to make use of the `shlex` python module as a good practice ?